### DISCUSSION TITLE
add skeleton placeholder

### DISCUSSION THREAD

**Discussion by patrickwasp** [zauberzeug/nicegui#3082] [Ideas / Feature Requests]
The QSkeleton is a component for displaying a placeholder preview of your content before you load the actual page data. It’s a nice way of informing the user of what to expect from the page before it is fully loaded and increases the perceived performance. It can be used to incrementally display information on screen as data is being fetched.

https://quasar.dev/vue-components/skeleton/

**Comment by falkoschindler** [comment]
Oh yes, that would be a great addition to the existing UI elements! And it shouldn't be too hard to implement.
Does anyone want to give it a go? 🙂

**Comment by falkoschindler** [comment]
This feature has been implemented in PR #3202 and released in version 1.4.28.
